src/billing_job.rs

```diff
@@ -1,46 +1,73 @@
 use std::sync::Arc;
 use std::time::Duration;
 
-use actix_web::web;
-use anyhow::{anyhow, Context};
-use tiny_keccak::{Hasher, Keccak};
 use crate::model::AppState;
+
+use actix_web::web::Data;
+use anyhow::{anyhow, Context, Error};
 use ethers::prelude::*;
 use k256::elliptic_curve::generic_array::sequence::Lengthen;
+use tiny_keccak::{Hasher, Keccak};
 
-pub async fn billing_scheduler(appstate: web::Data<AppState>,) -> Result<String, anyhow::Error> {
+pub async fn billing_scheduler(appstate: Data<AppState>,) -> Result<String, Error> {
     let provider = Provider::<Http>::try_from(&appstate.rpc)
-                                        .context("Unable to connect to the RPC")?
-                                        .interval(Duration::from_millis(1000));
-    let wallet: LocalWallet = appstate.operator_key.parse().context("Unable to parse operator private key")?;
+        .context("Unable to connect to the RPC")?
+        .interval(Duration::from_millis(1000));
+    let wallet: LocalWallet = appstate.operator_wallet_key
+        .parse()
+        .context("Unable to parse operator private key")?;
 
     let client = SignerMiddleware::new(provider, wallet);
-    let contract = Contract::new(appstate.contract.as_str()
-                                                                                        .parse::<Address>()
-                                                                                        .context("Unable to parse contract address")?, 
-                                                                        appstate.abi.clone(), 
-                                                                        Arc::new(client));
+    let contract = Contract::new(
+        appstate.contract.as_str()
+                    .parse::<Address>()
```

Comment:
maybe, you just 4 space indentation from the line above.

Reply:
Done!